decentraland/unity-explorer
# chore: download asset bundles for local scenes

## What does this PR change?

ONLY FOR DEBUGGING

Creates a HybridSceneApproach (LocalScene + Remote Asset bundle)
If the dev inputs the remote scene id and the content server correctly, they should be able to debug a scene with remote asset bundles. 
If the remote assets do not exist, regular pipeline will continue

## How to test the changes?

1. Launch the explorer and do the regular stuff. The common user should see no changes


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
